spyder_terminal/locale/es/LC_MESSAGES/spyder_terminal.po

```diff
@@ -1,37 +1,31 @@
-# -*- coding: utf-8 -*-
-# SOME DESCRIPTIVE TITLE.
-# Copyright (C) YEAR ORGANIZATION
-# FIRST AUTHOR <EMAIL@ADDRESS>, YEAR.
-#
 msgid ""
 msgstr ""
 "Project-Id-Version: spyder-terminal\n"
 "POT-Creation-Date: 2019-12-09 09:55-0500\n"
-"PO-Revision-Date: 2019-12-10 14:58\n"
-"Last-Translator: Gonzalo Peña-Castellanos (goanpeca)\n"
+"PO-Revision-Date: 2020-01-24 16:49\n"
+"Last-Translator: FULL NAME <EMAIL@ADDRESS>\n"
 "Language-Team: Spanish\n"
 "MIME-Version: 1.0\n"
 "Content-Type: text/plain; charset=UTF-8\n"
 "Content-Transfer-Encoding: 8bit\n"
 "Generated-By: pygettext.py 1.5\n"
 "Plural-Forms: nplurals=2; plural=(n != 1);\n"
-"X-Generator: crowdin.com\n"
 "X-Crowdin-Project: spyder-terminal\n"
 "X-Crowdin-Language: es-ES\n"
 "X-Crowdin-File: /master/spyder_terminal/locale/spyder_terminal.pot\n"
 "Language: es_ES\n"
 
 #: spyder_terminal/confpage.py:34
 msgid "Terminal shell"
-msgstr "Shell de terminal"
+msgstr "Shell de consola"
```

Comment:
I think we should name this "Consola del sistema". Or perhaps simply "Consola"?

@steff456, @goanpeca, what do you think?

Reply:
Consola del sistema ooo linea de comandos?

Reply:
I think "Consola del sistema" is simpler.

Reply:
Sure!